data/schema/schema_v1.json

```diff
@@ -615,7 +615,7 @@
                             },
                             "calculated": {
                               "type": "boolean",
-                              "description": "Value calculated from answers."
+                              "description": "Shows that answer is to be treated as result of calculation or not."
```

Comment:
This would read better as 'Indicates that the answer should be treated as the result of a calculation.'

Reply:
agreed, changed